src/components/Swap/Price.tsx

```diff
@@ -54,7 +54,13 @@ export default function Price({ trade, outputUSDC }: PriceProps) {
   const [exchangeRate, usdcPrice] = useTradeExchangeRate(trade, outputUSDC, defaultBase ? 'input' : 'output')
 
   return (
-    <TextButton color="primary" onClick={onClick}>
+    <TextButton
+      color="primary"
+      onClick={(e) => {
+        onClick()
+        e.stopPropagation()
```

Comment:
why the `stopPropagation` here 🤔 this feels sus

Reply:
this is because `Price` should handle clicks without the underlying `Toolbar` also receiving them. without this, when you click on `Price` to swap the exchange rate, the `Toolbar` will also expand.

is there a better way to achieve this?

Reply:
okay yeah i was having the same issue with my tooltips on mobile and was trying to think of a way around it

so `stopPropagation` would work but its considered a v risky maneuver as it prevents any other event handler on the page from being run. for example if you decide to add a click handler onto the whole page or if you want to do analytics tracking, it would prevent the event from reaching those callbacks. in the past these issues have been rly gnarly to debug so i would recommend not using `stopPropagation` if u can :3 

oneee alternative i can think of is to have all the clickable elements within the toolbar be `button` (i think it may already be like this?) and then detect whether or not the event was originated from a button that is nottt the current element if that makes sense

Reply:
another idea if we can't rely on the `button` type is to label other elements with a classname like `.clickable` and the handler can check to see if it originated from a component with `clickable` as a classname 🤔